priv/plan.graphql

```diff
@@ -74,7 +74,11 @@ query TripPlan(
         }
         to { ...PlaceInfo }
         transitLeg
-        trip { gtfsId }
+        trip { 
```

Comment:
```suggestion
        trip {
```

Github doesn't show this, but there's trailing whitespace at the end of this line.